### DISCUSSION TITLE
Small bug when making a storefront primary directly after creating it

### DISCUSSION THREAD

**Discussion by brahima** [swellstores/community#204] [Help] [answered]
Hello,

Just came across this when creating a new Storefront.

### Small bug when making a storefront primary directly after creating it

How to reproduce : 
- Go to Storefronts and create a new Storefront
<img width="1219" alt="image" src="https://user-images.githubusercontent.com/1575495/230500139-beafec6b-b0a5-4efb-8c96-bd2552ae9515.png">

- Put a name / a url and create the storefront (via the buttton 'Create Storefront')

<img width="1213" alt="image" src="https://user-images.githubusercontent.com/1575495/230500468-96d34364-4eda-4dd7-8bb3-0ddbbfa13329.png">

- Now go directly to the developers tab then'Actions' then 'Make primary', you'll get an error
<img width="1236" alt="image" src="https://user-images.githubusercontent.com/1575495/230500569-bb1fd88e-e6a8-4ed7-9926-29310d4751e5.png">

<img width="1301" alt="image" src="https://user-images.githubusercontent.com/1575495/230500596-4c4c087d-8ddf-493e-b1ef-7ac37f7b37bd.png">

If you leave the newly created storefront and come back or try the same action from the main menu, it works.

<img width="871" alt="image" src="https://user-images.githubusercontent.com/1575495/230501343-4946f3ed-8e34-4455-b440-a98ecd55426a.png">

Thanks

**Comment by SusannaSwell** [comment, marked as answer]
@brahima Thanks for the details. Checked on our end as well and can confirm that this is occurring. We will have this logged and let you know more as updates become available.

**Comment by SusannaSwell** [comment]
hello @brahima  this should be fixed now. Can you confirm if it's working fine?
